src/process/redux/userSlice.js

```diff
@@ -31,23 +30,20 @@ export const signUp = createAsyncThunk('user/signUp', async payload => {
 
 const userSlice = createSlice({
   name: 'user',
-  initialState: {},
-
+  initialState: { isLoggedIn: false },
   reducers: {},
-
   extraReducers: {
     [signIn.fulfilled]: (state, { payload }) => {
       Api.setHeaders(payload.headers);
       return { ...payload, isLoggedIn: true };
     },
 
     [signUp.fulfilled]: (state, { payload }) => {
-      Api.setHeaders(payload.headers);
       return { ...payload };
     },
 
-    [signOut.fulfilled]: (state, { payload }) => {
-      return { isLoggedIn: false };
+    [signOut.fulfilled]: state => {
+      return { state };
```

Comment:
Return the initial state, not an object containing the reducer's state.

Reply:
hope is right this time

Reply:
Define initial state into a const and use it in the createSlice function, and return it in here and in signUp